eras/shelley-ma/impl/src/Cardano/Ledger/ShelleyMA/AuxiliaryData.hs

```diff
@@ -84,53 +87,54 @@ deriving instance NoThunks (Script era) => NoThunks (AuxiliaryDataRaw era)
 
 instance NFData (Script era) => NFData (AuxiliaryDataRaw era)
 
-newtype MAAuxiliaryData era = AuxiliaryDataWithBytes (MemoBytes AuxiliaryDataRaw era)
+-- We call it AllegraTxAuxData because it is first used in the Allegra Era
```

Comment:
I don't think this comment is needed, this is a common pattern throughout the codebase.
```suggestion
```

Solid reasoning on choosing the name though!!!

Reply:
done